src/js/disability-benefits/components/TabNav.jsx

```diff
@@ -5,15 +5,11 @@ import TabItem from './TabItem';
 class TabNav extends React.Component {
   render() {
     return (
-      <div className="row">
-        <div className="medium-8 columns">
-          <ul className="va-tabs">
-            <TabItem tabpath="/your-claims/status" title="Status"/>
-            <TabItem tabpath="/your-claims/files" title="Files"/>
-            <TabItem tabpath="/your-claims/details" title="Details"/>
-          </ul>
-        </div>
-      </div>
+      <ul className="va-tabs">
```

Comment:
Tab nav seems have some spacing gaps (compared to rx):

![screen shot 2016-10-07 at 1 29 58 pm](https://cloud.githubusercontent.com/assets/634932/19199467/3e30d550-8c92-11e6-8dcc-cde203740fb1.png)